utils/index.js

```diff
@@ -208,3 +208,67 @@ module.exports.processMarkdown = require('markdown-it')({
   linkify: true,
   typographer: true
 })
+
+/**
+ * Returns an array of express Router instances
+ * or []
+ */
+module.exports.loadThemeRoutes = function (app) {
+  console.log('Loading configured theme routes...')
+  const config = require('../config')
```

Comment:
@starsinmypockets I think this should go to the top so can be re-used by other functions if needed